easydata.net/src/EasyData.Core/Services/EasyDataManager.cs

```diff
@@ -46,28 +46,108 @@ public EasyDataManager(IServiceProvider services, EasyDataOptions options)
 
         public async Task<MetaData> GetModelAsync(string modelId, CancellationToken ct = default)
         {
-            if (Model.Id !=modelId) {

```

Comment:
I think it's not nice that we go into somebody's project and change the code styles. Let's add an `.editorconfig` file and add a rule to allow block condition brackets to be on the same like. It will look something like this:

```editorconfig
[*.cs]
csharp_new_line_before_open_brace = accessors,types,methods,properties,indexers,events,anonymous_methods,anonymous_types,object_collection_array_initalizers,lambdas,local_functions
```